pkg/ctl/topic/set_retention.go

```diff
@@ -27,12 +27,12 @@ import (
 func SetRetentionCmd(vc *cmdutils.VerbCmd) {
 	desc := cmdutils.LongDescription{}
 	desc.CommandUsedFor = "Set the retention policy for a topic"
-	desc.CommandPermission = "This command requires tenant admin permissions."
+	desc.CommandPermission = "This command requires tenant admin permissions and topic-level policy should be enabled in brokers"
```

Comment:
I think the CommandPermission is more about the auth related things. The topic-level policy is a configuration of the broker. User should be aware of that when they setup cluster.

Reply:
Sounds reasonnable.  I added some hint mesage for the err code so if users miss to setup the config and running the topic set-retention command, they will get enough info to update the config.